OpenImageIO_v2_4::ImageBuf::IteratorBase::operator++(int)' definition is marked dllimport

When doing a shared build of versions 2.4.4.1 and 2.4.4.2 on Windows with MinGW-w64 building fails with the following error message:
```
src/include/OpenImageIO/imagebuf.h:1302:23: error: function 'void OpenImageIO_v2_4::ImageBuf::IteratorBase::operator++(int)' definition is marked dllimport`
```
Build configuration with CMake looks like this:
```bash
C_INCLUDE_PATH=$MINGWPREFIX/ffmpeg5/include${C_INCLUDE_PATH:+:$C_INCLUDE_PATH} CPLUS_INCLUDE_PATH=$MINGWPREFIX/ffmpeg5/include${CPLUS_INCLUDE_PATH:+:$CPLUS_INCLUDE_PATH} PKG_CONFIG_PATH=$MINGWPREFIX/ffmpeg5/lib/pkgconfig${PKG_CONFIG_PATH:+:$PKG_CONFIG_PATH} PATH=$MINGWPREFIX/ffmpeg5/bin:$PATH:$MINGWPREFIX/qt/bin QTDIR=$MINGWPREFIX/qt QMAKESPEC=$MINGWPREFIX/qt/mkspecs/win32-g++ cmake.exe -Wno-dev -GNinja -DCMAKE_INSTALL_PREFIX:PATH=$INSTALLPREFIX -DCMAKE_BUILD_TYPE:STRING=Release -DBUILD_SHARED_LIBS:BOOL=ON -DCMAKE_CXX_STANDARD:STRING=14 -DUSE_QT:BOOL=ON -DUSE_PYTHON:BOOL=OFF -DBUILD_MISSING_DEPS:BOOL=OFF -DBUILD_MISSING_FMT:BOOL=OFF -DCODECOV:BOOL=OFF -DOIIO_BUILD_TESTS:BOOL=OFF -DSTOP_ON_WARNING:BOOL=ON -DJPEGTurbo_ROOT:PATH=$MINGWPREFIX/share/libjpeg-turbo -DPKG_CONFIG_EXECUTABLE:FILEPATH=$(which pkg-config.exe) -DCMAKE_CXX_FLAGS:STRING="-Wno-error=deprecated-declarations" -DCMAKE_SHARED_LINKER_FLAGS:STRING="-Wl,--as-needed -lws2_32" -DCMAKE_MODULE_LINKER_FLAGS:STRING="-Wl,--as-needed -lws2_32" -DBUILD_TESTING:BOOL=OFF -S. -Bbuild_shared
```

Building succeeds when in `src/include/OpenImageIO/imagebuf.h` the line
```c
        void OIIO_API operator++(int) { ++(*this); }
```
is replaced with
```c
        inline void operator++(int) { ++(*this); }
```

I'm not sure this is the right solution, but some kind of fix is needed.

--- a/src/include/OpenImageIO/imagebuf.h
+++ b/src/include/OpenImageIO/imagebuf.h
@@ -1299,7 +1299,7 @@ public:
             pos(m_x, m_y, m_z);
         }
         /// Increment to the next pixel in the region.
-        void OIIO_API operator++(int) { ++(*this); }
+        void operator++(int) { ++(*this); }
 
         /// Return the iteration range
         ROI range() const

